core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -472,8 +472,8 @@ object KafkaConfig {
   val ControlledShutdownRetryBackoffMsDoc = "Before each retry, the system needs time to recover from the state that caused the previous failure (Controller fail over, replica lag etc). This config determines the amount of time to wait before retrying."
   val ControlledShutdownEnableDoc = "Enable controlled shutdown of the server"
   /** ********* Consumer coordinator configuration ***********/
-  val ConsumerMinSessionTimeoutMsDoc = "The minimum allowed session timeout for registered consumers"
-  val ConsumerMaxSessionTimeoutMsDoc = "The maximum allowed session timeout for registered consumers"
+  val ConsumerMinSessionTimeoutMsDoc = "The minimum allowed session timeout for registered consumers. Shorter timeouts lead to more consumer heartbeating which can overwhelm broker resources."
```

Comment:
"Shorter timeouts leader to more sensitive failure detection at the cost of more frequent consumer heartbeating required, which can overwhelm broker resources."